src/components/asthma/views/AsthmaActionPlanView/AsthmaActionPlanView.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Layout, NavigationBar } from '../../../presentational';
+import { AsthmaActionPlanManager } from '../../components';
+import { AsthmaActionPlan } from '../../model';
+
+export interface AsthmaActionPlanViewProps {
+    colorScheme?: 'light' | 'dark' | 'auto';
+    previewState?: 'loading' | 'loaded without action plan' | 'loaded with action plan'
+    learnMoreUrl: string;
+    onViewActionPlan: (actionPlan: AsthmaActionPlan) => void;
```

Comment:
What does this typically invoke?  Seems like it is always just viewing the file no...